packages/js/manifests/polywrap/src/formats/polywrap/index.ts

```diff
@@ -35,18 +40,20 @@ export const PolywrapManifestSchemaFiles: Record<string, string> = {
   "0.1.0": "formats/polywrap/0.1.0.json",
   "0.2.0": "formats/polywrap/0.2.0.json",
   "0.3.0": "formats/polywrap/0.3.0.json",
+  "0.4.0": "formats/polywrap/0.4.0.json",
```

Comment:
Hey @cbrzn, check the CHANGELOG.md file, we've actually added version `0.3.0` within toolchain version `0.10.0`, so we don't need to increment this to `0.4.0` since `0.3.0` is still in a pre-release state.

Here's the entry from the CHANGELOG for your reference:
![image](https://user-images.githubusercontent.com/5522128/213058033-07fad632-ee33-4e64-ba11-4b745c64892c.png)

Reply:
FYI I'm working on making this change, and the removal of extensions as described above in another branch, will PR into this branch soon. Will report back soon.